.github/workflows/ci.yml

```
@@ -18,23 +20,40 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-
+      
+    # Step to login to DockerHub
       - name: Log in to Docker Hub
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
+          username: ${{ secrets.DOCKERHUB_USERNAME}}
+          password: ${{ secrets.DOCKERHUB_TOKEN}}
+      
+    # Step to login to GitHub Container Registry
+      - name: Log in to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor}}
+          password: ${{ secrets.GITHUB_TOKEN}}
+      
+    # Extract metadata for the Docker images using the semver
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ vars.DOCKER_ORG }}/ecs-deploy
+          images: |
+            ${{ vars.DOCKER_ORG }}/ecs-deploy  # the name needs to be static "ecs-deploy"
+            ghcr.io/${{ github.repository_owner }}/ecs-deploy
+          tags: |
+           type=semver,pattern={{version}}
```

keep an eye on consistent indentation, also add the major.minor version
```suggestion
            type=semver,pattern={{version}}
            type=semver,pattern={{major.minor}}
```